README.md

```diff
@@ -21,18 +21,21 @@ The simplest configuation, `DEMO`, assumes you have a single meter strip of 144
 
 ## Getting Started
 I recommend you do the following:
+- Copy include/secrets.example.h to include/secrets.h; Set your WiFi SSID and password in include/secrets.h.
 - Build the source code.  In particular, build the `DEMO` configuration. Some pointers on what's needed to do this can be found [below](#build-pointers).
 - Upload the resultant binary to the ESP32
 - Connect PIN5 and GND and 5V of a WS2812B strip to the ESP32
 - Provide an adequate power source for the LEDs and ESP32
 - Enjoy the pretty lights
-- Start enabling features in the `globals.h` file like WiFi and WebServer.  Set your WiFi SSID and password in secrets.h.
+- Start enabling features in the `globals.h` file like WiFi and WebServer.
 - Connect to the ESP32's web user interface with a browser to its IP address
 
 ## Wifi Setup
-Set your WiFi SSID and password in include/secrets.h.
- - You can prevent git from tracking changes in this file by telling it to assume the file is unchanged. <br/>
-   `git update-index --assume-unchanged include/secrets.h`
+Ensure your WiFi SSID and password are set in include/secrets.h.

```

Comment:
I think we should add something after the "Ensure your WiFi SSID...", line, like: Please do make sure you set them in `include/secrets.h`, **not** `include/secrets.example.h`.

I'd really like to prevent someone not paying enough attention at exactly that moment, and later accidentally pushing their network details in a PR.

Similarly, I think we should include a comment in (now) include/secrets.example.h to instruct people to copy the file and only THEN add their credentials, just above the defines. Something like:
```cpp
// NOTE: do NOT enter your network details in this file (secrets.example.h)! Instead, copy this file to secrets.h, and set the below defines in that file.`
```

Reply:
I've made the requested changes.

Reply:
Thank you. Also for putting this PR together, I think this is a much more robust approach, overall.

Reply:
No problem, I wasn't really happy with how this was working. 

Also, on a side note.  I will submit a PR here shortly to significantly speed up the CI. (It will pre-download the base project dependencies to take advantage of the lib folder)

Reply:
Great, looking forward to it!